src/main/kotlin/Graph.kt

```diff
@@ -0,0 +1,68 @@
+import Graph.Companion.STOCKHOLM
+import Graph.Companion.UPPSALA
+
+class Graph {
+    fun f() {
+        return
+        println() // unreachable
+    }
+
+    companion object {
+        const val STOCKHOLM = "ID_STOCKHOLM"
+        const val UPPSALA = "ID_UPPSALA"
+    }
+
+    val user: User? = null
+
+    fun findShortestPath(node1: Node, node2: Node): Int {
+        return 0
+        val x = 3
+        println() // unreachable
+    }
+
+    fun getNodeById(id: String): Node {
+
+        // TODO: ?
+
+        @Suppress("UnsafeCallOnNullableType")
+        val firstName = user!!.firstName
+
+        return Node()
+    }
+}
+
+class Node
+
+class User {
+    val firstName = ""
+
+    fun fun1() {}
+    fun fun2() {}
+    fun fun3() {}
+    fun fun4() {}
+    fun fun5() {}
+    fun fun6() {}
+    fun fun7() {}
+    fun fun8() {}
+    fun fun9() {}
+    fun fun10() {}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/51112b832939022334567e2f7c8f1966949562a4/src/main/kotlin/Graph.kt#L48"></span>This empty block of code can be removed.</td>
    </tr>
  </tbody>
</table>